lib/defaults.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var path = require('path');
+
+var DEFAULTS = {
+  CRON_TIMING : '30 30 1 * * 3',
+  SOURCE_FILE_NAME : 'sources.json',
+  TARGET_DIR_NAME : 'db',
+  TARGET_FILE_NAME : 'cities-db.mmdb',
+  TIMEZONE : 'America/Los_Angeles'
+};
+
+DEFAULTS.DEFAULT_TARGET_DIR_PATH = path.join(__dirname, '..', DEFAULTS.TARGET_DIR_NAME);
+DEFAULTS.DEFAULT_DB_PATH = path.join(DEFAULTS.DEFAULT_TARGET_DIR_PATH, DEFAULTS.TARGET_FILE_NAME);
+DEFAULTS.DEFAULT_SOURCE_FILE_PATH = path.join(__dirname, '..', DEFAULTS.SOURCE_FILE_NAME);
```

Comment:
FWIW, you may need/want to tweak the `DEFAULTS_` prefix in these last 3 items:

```json
{
  "CRON_TIMING": "30 30 1 * * 3",
  "SOURCE_FILE_NAME": "sources.json",
  "TARGET_DIR_NAME": "db",
  "TARGET_FILE_NAME": "cities-db.mmdb",
  "TIMEZONE": "America/Los_Angeles",
  "DEFAULT_TARGET_DIR_PATH": "/Users/pdehaan/dev/tmp/del/fxa-geodb/db",
  "DEFAULT_DB_PATH": "/Users/pdehaan/dev/tmp/del/fxa-geodb/db/cities-db.mmdb",
  "DEFAULT_SOURCE_FILE_PATH": "/Users/pdehaan/dev/tmp/del/fxa-geodb/sources.json"
}
```

The variables are still [a semi-verbose] `DEFAULTS.DEFAULT_SOURCE_FILE_PATH`;